packages/trader/src/Modules/Trading/Components/Form/TradeParams/amount-mobile.jsx

```diff
@@ -65,28 +67,46 @@ const Basis = ({
     };
 
     return (
-        <div className='trade-params__amount-keypad'>
-            <Numpad
-                value={selected_basis}
-                format={formatAmount}
-                onSubmit={setBasisAndAmount}
-                currency={currency}
-                min={min_amount}
-                is_currency
-                render={({ value: v, className }) => {
-                    return (
-                        <div className={className}>
-                            {parseFloat(v) > 0 ? <Money currency={currency} amount={v} should_format={false} /> : v}
-                        </div>
-                    );
-                }}
-                reset_press_interval={450}
-                reset_value=''
-                pip_size={user_currency_decimal_places}
-                onValidate={validateAmount}
-                submit_label={localize('OK')}
-                onValueChange={onNumberChange}
-            />
+        <div className='trade-params__strike'>
+            {contract_type === 'vanilla' && (
+                <section className='trade-container__stake-field'>
+                    <div className='trade-container__stake-field--min'>
+                        <Text size='xxs'>Min. Stake</Text>
+                        <Text size='xxs'>
+                            {stake_boundary.min_stake} {currency}
+                        </Text>
+                    </div>
+                    <div className='trade-container__stake-field--max'>
+                        <Text size='xxs'>Max. Stake</Text>
+                        <Text size='xxs'>
+                            {stake_boundary.max_stake} {currency}
```

Comment:
localization

Reply:
Not needed as the values are numeral and currency unit